utils/http/http_test.go

```diff
@@ -32,7 +34,10 @@ type Label struct {
 }
 
 func TestGetRequest(t *testing.T) {
-	data, err := GetRequest("http://www.mocky.io/v2/589bb17d100000701266e5e1")
+	var ctx context.Context
```

Comment:
single use variable. we can directly send ctx.Background() in the GetRequest func